### ISSUE TITLE
Addressbook/Bookmark feature

### ISSUE THREAD

**Issue by maxxer** [novnc/noVNC#1227]
It would be great to have a bookmarks/address book feature. 

Having a list of VNC servers, with or without stored password, can be very handy.

**Comment by samhed** [comment]
Agreed!

**Comment by CendioOssman** [comment]
I'm not sure how this would work though? In most cases we need a websocket proxy in between, that in many cases also handles the multiplexing.

So the bookmarks would only be for websocket proxies, not VNC servers. I think there might be a risk for confusion, like we had when we had a host field front and center on the page.

**Comment by samhed** [comment]
Yes, it wouldn't straight up be a list of VNC servers since a proxy is required inbetween, many popular VNC servers includes a proxy though (QEMU and x11vncserver etc.).

You could also imagine a setup where you have a webbserver hosting noVNC, and behind it you have multiple different proxy+VNC server combinations. In such cases this feature would be useful. I'm imagining something quite simple with a dropdown.

PS:
To avoid confusion, we should probably rename the label for this in the settings. It's currently called "host", I believe "WebSocket proxy" would be better.
